src/librustdoc/html/render/mod.rs

```diff
@@ -1031,11 +1031,11 @@ themePicker.onblur = handleThemeButtonsBlur;
     // with rustdoc running in parallel.
     all_indexes.sort();
     {
-        let mut v = String::from("var searchIndex = JSON.parse('{\\\n");
+        let mut v = String::from("var searchIndex = JSON.parse(\"{\\n");
```

Comment:
```suggestion
        let mut v = String::from("var searchIndex = JSON.parse('{\\\n");
```

This needs to stay using single quotes because the strings being inserted here don't currently have the double quotes escaped. Also this was removing a backslash that I can only assume was a typo.

This is the cause of the CI failure.